docs/kubernetes/distributions/rke1.md

```diff
@@ -0,0 +1,67 @@
+
+Since RKE2 uses containerd instead of Docker Engine (which is supported), it can still support Windows hosts.
+
+> **Note**: While it is possible to deploy  [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd), a [shim](https://en.wikipedia.org/wiki/Shim_(computing)) that allows Kubernetes to communicate with Docker via the expected [Container Runtime Interface (CRI)](https://kubernetes.io/docs/concepts/architecture/cri/), to use Docker Engine as the container runtime on a Windows host, this conflicts with Microsoft's guidance since Microsoft does not recommend using `cri-dockerd` and has deprecated Docker on Windows hosts; this is the core reason why RKE1 Windows reached End of Life (EOL).
+
+## How does k3s/RKE2 differ from RKE1?
+
+This newer distributions avoid some of the pitfalls baked into the design of RKE1; notable difference include, but are not limited to:
+
+1. RKE1 relies on an **"externally" managed [Docker](https://www.docker.com)** (or Docker Engine) running on the host as the underlying [container runtime](https://opensource.com/article/21/9/container-runtimes), whereas RKE2 provides an **embedded [containerd](https://containerd.io/)** that comes as part of the RKE2 installation process itself. This reduces a lot of user bugs around invalid host [dockerd](https://docs.docker.com/engine/reference/commandline/dockerd/) configurations since RKE2 retains **full control over the configuration of the container runtime** it ships within the RKE2 binary itself.
+2. RKE1 has an **imperative** model of installation / upgrade relying on an **external** host (i.e. your computer or the Rancher server) running commands like `rke up` to get the nodes up and running. This requires the external host to be **capable of SSHing into each of the nodes**. RKE2 follows a **declarative** model of installation / upgrades that happens **directly on the host**. To do this, each host registers RKE2 as a **[systemd unit](https://www.digitalocean.com/community/tutorials/what-is-systemd)** that watches for changes to a configuration file to perform configuration updates. This makes RKE2 more suitable for declarative provisioning solutions like **booting up nodes via [cloud-init](https://cloud-init.io) or [Terraform](https://www.terraform.io)**.
+3. RKE1 makes **Docker client calls** to deploy system components (kubelet, kube-proxy, etc.) as **Docker containers**. You can debug RKE1 via commands like `docker ps` or `docker log`. RKE2 deploys system components as **static pods managed by kubelet itself**. You can debug RKE2 via commands like `crictl ps` or `crictl logs` (since containerd is the underlying container runtime for RKE2).
+4. RKE1 supports certain CNI plugins (Canal, Flannel, Calico, and Weave), whereas RKE2 supports a different set of CNI plugins (Cilium, Calico, Canal and Multus) for Linux clusters. For Windows clusters, RKE supports Flannel and RKE2 supports Canal.
```

Comment:
we should add that when it's no longer experimental

Reply:
Because this is not officially supported by rke2 we will not be adding this to the docs, but I'll leave this comment for future reference when it is an officially supported cni